consumer/spring-cassandra-consumer/src/main/java/org/springframework/cloud/fn/consumer/cassandra/CassandraConsumerConfiguration.java

```diff
@@ -170,9 +174,13 @@ protected List<List<Object>> transformPayload(Object payload) {
 							Object value = entity.get(column);
 							if (value instanceof String string) {
 								if (this.dateFormat.looksLikeISO8601(string)) {
-									synchronized (this.dateFormat) {
+									try {
+										this.dateLock.lock();
```

Comment:
The `lock()` call is supposed to be outside of the `try` block. See `Lock` JavaDocs.